tests/__init__.py

```diff
@@ -1 +1,6 @@
 #
+try:
```

Comment:
@ColinTalbert is there a reason for putting this here rather than what we already have in `test_tool.py`?